### ISSUE TITLE
salt-run doc.execution fails with AttributeError

### ISSUE THREAD

**Issue by martinhoefling** [saltstack/salt#25016]
```
root@ubuntu:~# salt-run doc.execution
Exception occurred in runner doc.execution: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/salt/client/mixins.py", line 317, in low
    data['return'] = self.functions[fun](*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/salt/runners/doc.py", line 76, in execution
    i = itertools.chain.from_iterable([i.items() for i in six.itervalues(docs)])
AttributeError: 'int' object has no attribute 'items'
```

Debian / Ubuntu with latest packages:
```
root@minastirith:~# salt   --versions-report
                  Salt: 2015.5.0
                Python: 2.7.9 (default, Mar  1 2015, 12:57:24)
                Jinja2: 2.7.3
              M2Crypto: 0.21.1
        msgpack-python: 0.1.10
          msgpack-pure: Not Installed
              pycrypto: 2.6.1
               libnacl: Not Installed
                PyYAML: 3.11
                 ioflo: Not Installed
                 PyZMQ: 14.4.0
                  RAET: Not Installed
                   ZMQ: 4.0.5
                  Mako: 1.0.0
 Debian source package: 2015.5.0+ds-1~bpo70+1
```

**Comment by martinhoefling** [comment]
See pull request #25020

**Comment by cachedout** [comment]
Closed by #25020
